0MI0800304/7.03/ConsoleApplication1/problem123.cpp

```diff
@@ -0,0 +1,98 @@
+#include <iostream>
+#include <fstream>
+//problem 1
+void writeData() {
+	std::ofstream file("result.txt");
```

Comment:
Mmm the name of the file should be an argument to the function, hard-coding it is everything but nice :P